setup.sh

```diff
@@ -57,6 +57,8 @@ touch discovery-provider/.env
 
 # setup service
 if [[ "$1" != "" ]]; then
+	audius-cli auto-upgrade "$1" || true
```

Comment:
I do not believe we should do this

It is against the spirit of decentralization and obfuscates the fact that we are adding auto-upgrade